AutoGear.lua

```diff
@@ -2574,8 +2576,16 @@ end
 function AutoGearGetPawnScaleName()
 	local _, _, ClassID = UnitClass("player")
 
-	local spec = AutoGearGetSpec()
+	-- Get spec inc overeide if enabled
+	local class, spec = AutoGearGetClassAndSpec()
 
+	-- Try to find the exact match for spec name in Pawn
+	for ScaleName, Scale in pairs(PawnCommon.Scales) do
+		if spec == ScaleName then
+			return ScaleName
+		end
+	end
+	
```

Comment:
This section isn't documented clearly in the options menu.  For overriding Pawn scale, we should provide a list of Pawn scales that exist in a dropdown instead.  I plan to do that soon, but if you want to implement that, it would be very helpful.

Also, with this logic, it will only match "Protection" as the Pawn scale name, and will ignore the class, so the scale would have to be named just "Protection", which isn't documented.